fuel/schemes.py

```diff
@@ -232,3 +232,48 @@ def get_request_iterator(self):
         indices = list(self.indices)
         self.rng.shuffle(indices)
         return iter_(indices)
+
+
+def k_fold_cross_validation(scheme_class, num_examples, k, strict=True,
+                            **kwargs):
+    """Return pairs of schemes to be used for cross-validation.
+
+    Parameters
+    ----------
+    scheme_class : class
+        The type of the returned schemes. The constructor is called with an
+        iterator and `**kwargs` as arguments.
+    num_examples : int
+        The number of examples in the datastream.
+    k : int
+        The number of folds to return.
+    strict : bool, optional
+        If `True`, enforce that `num_examples` is divisible by `k` and so,
```

Comment:
Can you also specify what happens when it's `False`? I know it's specified in the `Return` section, but I prefer it to be repeated than missed by users.